test/integration/keys.spec.ts

```diff
@@ -0,0 +1,46 @@
+import { testClient } from "hono/testing";
+import { tsoaApp } from "../../src/app";
+import { getAdminToken } from "../../integration-test/helpers/token";
+import { getDb } from "../../src/services/db";
+import SQLite from "better-sqlite3";
+import { SqliteDialect } from "kysely";
+import createAdapters from "../../src/adapters/kysely";
+import { migrateToLatest } from "../../migrate/migrate";
+import { getCertificate } from "../../integration-test/helpers/token";
+
+describe("tenants", () => {
+  it("should add a new tenant", async () => {
```

Comment:
```suggestion
describe("keys", () => {
  it("should add a new key", async () => {
```
:question:  It is adding a tenant... but the spec is called keys...